### PR TITLE
Bump svg-dep, add builtin write_labels

### PR DESCRIPTION
**Pull request by tlambert03**
# Description
This bumps the napari-svg dependency to include the "trylast" marker from the last release on images and labels and the multiwriter.  So images will be written as tiff by default, instead of svg.  I added a test to confirm the load order.  We can revisit this in the future when we have persistence.

I also noticed that we didn't have a builtin `write_labels`... so I added one (it's the same as write_image except for dtype), added it to the tests, and  fixed a regression in the "guessing" of labels type with uint32/64

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by sofroniewn — APPROVED** [review]
LGTM! Nice additions

File: napari/plugins/_builtins.py [hunk 1]
```diff
@@ -69,6 +69,30 @@ def napari_write_image(path: str, data: Any, meta: dict) -> Optional[str]:
...
+    meta : dict
+        Image metadata.
+
+    Returns
+    -------
+    path : str or None
+        If data is successfully written, return the ``path`` that was written.
+        Otherwise, if nothing was done, return ``None``.
+    """
+    return napari_write_image(path, np.asarray(data, dtype='uint32'), meta)
```
Comment (jni) [inline napari/plugins/_builtins.py]: Hmm, dunno about hardcoding the dtype, vs specifying a minimum of int32. e.g. if labels are uint64, I don't think we want to downcast!

Comment (jni) [reply napari/plugins/_builtins.py]: I suggest `dtype = data.dtype if data.dtype.itemsize >= 4 else np.uint32`. (You could be even fancier with testing for negative values, but I'm not as concerned about that.)

Comment (tlambert03) [reply napari/plugins/_builtins.py]: I did that after some trial and error with Fiji ... it seemed that even with bioformats, the uint64 wasn’t working (image was entirely 0s).  But I agree it’s not nice to downcast... but slightly worried about people trying to open in Fiji

Comment (tlambert03) [reply napari/plugins/_builtins.py]: implemented your suggestion

Comment (jni) [reply napari/plugins/_builtins.py]: Ah, interesting. Well, if you want, I'd be ok with checking that the max value fits in uint32. But that will be hard with dask-y things.

Comment (jni) [reply napari/plugins/_builtins.py]: anyway, happy with a merge as its stands

Comment (tlambert03) [reply napari/plugins/_builtins.py]: Yeah let’s worry about that once someone gets confused with cross compatibility

**Pull request merged**